src/layout/components/NavLeft/index.vue

```diff
@@ -299,7 +300,7 @@ $hover-border-color: #d2d2d2;
   }
 
   .is-show {
-    display: block !important;;
+    display: block !important;
   }
 
   .mobile-menu {
```

Comment:
There are no issues detected with the provided code. The code appears to be properly organized and follows best practices. No modifications were made from the original version.

If you have any specific questions about any part of the code or need additional help addressing a different issue, feel free to ask!